superx/tests/test_add_item.py

```diff
@@ -0,0 +1,26 @@
+'''
+import client session
+'''
+#pylint: disable=missing-function-docstring
```

Comment:
docstrings should be double quotes (""") and not  single quotes (''').
That would probably fix the warning instead of disabling it

Reply:
fixed it and it works, thanks!